### PR TITLE
feat(replay): Include a link to view replays in new-issue slack messages

### LINKED ISSUE #51934: Update slack messages to include replay info

### PR DESCRIPTION
**Pull request by ryan953**
The slack message without, and then with the replay link (the triangle is included in the linked text):

SCR-20230720-mrvv

I experimented with different "icons", using an image doens't seem to work. Also, different unicode triangles will automatically trigger slack to convert it into an emoji. I didn't love that, it looks like this:
SCR-20230720-mvhp

Fixes https://github.com/getsentry/sentry/issues/51934

### CODE REVIEW

**Comment by ceorourke** [comment]
Looks good! Can you add a test? Maybe in https://github.com/getsentry/sentry/blob/master/tests/sentry/integrations/slack/test_notifications.py

**Comment by ceorourke** [comment]
I think it looks fine as is and I'm not sure what you tried, but the sentry icon is built like this: https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/message_builder/base/base.py#L78

**Comment by ryan953** [comment]
> I think it looks fine as is and I'm not sure what you tried, but the sentry icon is built like this: https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/message_builder/base/base.py#L78

@ceorourke I was trying to inline a little logo into the body text. I figured the text is markdown, so it might work, but slack didn't seem to parse it so everything i tried ended up with the full url being shown in there. I switched to the unicode character which seems ok. I wasn't working off any design, so if anyone had an issue I could drop that too.

**Comment by jas-kas** [comment]
@ryan953 Is it possible for the 'play button triangle' to be closer in size to the small Sentry logo? 

Otherwise looks great :)

**Review by jas-kas — APPROVED** [review]

**Comment by ryan953** [comment]
> @ryan953 Is it possible for the 'play button triangle' to be closer in size to the small Sentry logo?
> 
> Otherwise looks great :)

@jas-kas that's tough, right now it's a unicode character, not an image or anything
I was even thinking of dropping that part if people didn't love it.

**Review by ceorourke — APPROVED** [review]

File: tests/sentry/integrations/slack/test_message_builder.py [hunk 1]
```diff
@@ -229,6 +229,24 @@ def test_build_performance_issue(self):
         assert attachments["fallback"] == f"[{self.project.slug}] N+1 Query"
         assert attachments["color"] == "#2788CE"  # blue for info level
 
+    def test_build_replaye_issue(self):
```
[the remark this answers is not in the document]
Comment (ryan953) [reply tests/sentry/integrations/slack/test_message_builder.py:232]: lol, what if we are rebranding!?

[the remark this answers is not in the document]
Comment (ryan953) [reply tests/sentry/integrations/slack/test_message_builder.py:232]:
```suggestion
    def test_build_replay_issue(self):
```

**Pull request merged**